welcome/index.adoc

```diff
@@ -55,6 +55,18 @@ ifdef::openshift-online,openshift-aro[]
 Start with **xref:../architecture/architecture.adoc#architecture-overview-architecture[Architecture]**.
 endif::[]
 
+ifdef::microshift[]
+Start with xref:../microshift_getting_started/microshift-understanding.adoc#microshift-understanding[Understanding {product-title}] and
+xref:../microshift_install/microshift-install-rhel-for-edge.adoc#microshift-install-rhel-for-edge[Installing].
+Then, see the
+xref:../microshift_release_notes/microshift-4-12-release-notes.adoc#microshift-4-12-release-notes[release notes].
+
+[IMPORTANT]
+====
+Red Hat will not provide or support an update or upgrade path from Developer Preview or Technology Preview versions to later versions of {product-title}. A new installation will be necessary.
```

Comment:
This sentence is really complex with the amount of "ors" there are in the sentence (3).

Could we simplify this a bit and keep the same sentiment? Also, is it necessary to even mention TP right now since it's not even been provided as TP yet?

```suggestion
Red Hat will not support updating from Developer Preview versions to later versions of {product-title}. A new installation will be necessary.
```

Reply:
I think this edit isn't as explicit as we need, but I will change that third "or" to an "and", split the difference a bit.